src/functions.js

```diff
@@ -583,6 +587,9 @@ defineFunction(["\\over", "\\choose"], {
         case "\\choose":
             replaceWith = "\\binom";
             break;
+        case "\\atop":
+            replaceWith = "\\\\atopfrac";
+            break;
```

Comment:
Testing is great, creating tests is nonzero work, updating tests is nonzero work, running tests takes time (Jasmine tests are much faster). KaTeX’s behavior is not always “right”, I've had to update a lot of screenshotter images because previously-encoded behavior was wrong, and I'm sure as time goes on that will be repeated. But the occasional screenshotter test was valuable to catch a problem (StyleSwitching in particular).

I find tests for edge cases more important than tests for typical cases, so just adding a screenshotter test for, say, “{1 \atop 2}” (which is what I did) is low-value. When a bug is fixed, particularly a tricky one, it's usually important to add a test.